src/frontend/components/actions/bulk-delete.tsx

```diff
@@ -66,16 +66,20 @@ const BulkDelete: React.FC<ActionProps & AddNoticeProps> = (props) => {
     })
   }
 
+  const cssTag = `${resource.id}-bulk-delete-content`
+  const tableTag = `${resource.id}-bulk-delete-table`
+  const footerTag = `${resource.id}-bulk-delete-footer`
+
   return (
     <>
-      <DrawerContent>
+      <DrawerContent data-css={cssTag}>
```

Comment:
Why `cssTag` and not `contentTag`? It's the same in other files, one of the tags is always `cssTag` 🤔

Reply:
+ why do you hardcode `bulk-delete` instead of doing `${action.name}`?

Reply:
done